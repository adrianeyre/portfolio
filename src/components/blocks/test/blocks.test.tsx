import { shallow } from 'enzyme';

import IBlocksProps from '../interface/blocks-props';
import Blocks from '../blocks';

describe('Blocks', () => {
	const mockRouteProp: any = jest.fn();

	const defaultProps: IBlocksProps = {
		scrollToAnchor: mockRouteProp,
		showModal: mockRouteProp,
	};
	
	it('Should render correctly', () => {
		const blocksComponent = shallow(<Blocks { ...defaultProps }/>);

		expect(blocksComponent).toMatchSnapshot();
	});
});
