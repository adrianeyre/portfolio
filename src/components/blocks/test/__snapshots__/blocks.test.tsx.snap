// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blocks Should render correctly 1`] = `ShallowWrapper {}`;
